.github/workflows/lint.yml

```diff
@@ -24,3 +24,4 @@ jobs:
         run: |
           black --check setup.py
           black --check modules/ALFA/ALFA.py
+          black --check modules/ALFA/custom_code.py
```

Comment:
This line is OK but could you please point me to the docs where they say that there has to be an additional file with the hooks?

Reply:
Ofcourse, two files are not necessary, I was just following `example-plugin`, deleted `custom_code.py` and transferred contents to `ALFA.py`